src/CMakeLists.txt

```diff
@@ -159,6 +159,13 @@ else()
   target_compile_definitions(cellml PUBLIC ${LIBXML2_DEFINITIONS})
 endif()
 
+# Prevent superfluous warnings from ranlib about libcellml.a having no symbols
+# on macOS.
+if(APPLE)
+  set(CMAKE_CXX_ARCHIVE_CREATE "<CMAKE_AR> Scr <TARGET> <LINK_FLAGS> <OBJECTS>")
```

Comment:
Is this line required?  Does it change anything from the default?

Reply:
Yes, I have no idea why, but if you don't set `CMAKE_CXX_ARCHIVE_CREATE` and `CMAKE_CXX_ARCHIVE_FINISH` then you get two warnings. If you only set one of them then you still get one warning. It's only when you set both that no warning is generated.